src/accessibility/WCAG-2.2/index.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Changes to the Design System to meet WCAG 2.2
+description: Outlines the current principles and work needed to improve the accessibility of the GOV.UK Design System
+section: Accessibility
+layout: layout-pane.njk
+showPageNav: true
+order: 2
+---
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+W3C owns and manages the [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/).
+
+In 2023, the Design System team assessed and updated the GOV.UK Design System to comply with the new criteria and recommendations given in WCAG 2.2.
+
+Any guidance we provide is to make sure teams using the Design System are aware of the changes and can make the necessary adjustments to their services.
+
+### Make sure your service meets the new WCAG 2.2 criteria
+
+WCAG 2.2 was published in October 2023. You’ll need to comply with the new criteria no later than October 2024. See more about [Meeting government accessibility requirements](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag) in the GOV.UK Service Manual.
+
+### What you need to do
+
+1. Revisit the [Government Digital Service (GDS) guidance](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps) on accessibility is and why your service needs to invest in it
+2. Read [What’s new in WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/) to understand the new criteria your service will need to comply with
+3. Ask your wider organisation for support to prepare for work needed to meet WCAG 2.2
+4. Review the lists on this page of affected components and patterns in the Design System
```

Comment:
Perhaps add a list item for:
> 4. Update your service to use GOV.UK Frontend version 5.0 or higher

Or something like that. Just noticed it's not mentioned anywhere in this WCAG 2.2 page.